addons/mail/static/src/core/common/message_action_menu_mobile.js

```diff
@@ -13,6 +13,8 @@ export class MessageActionMenuMobile extends Component {
         "messageEdition?",
         "messageToReplyTo?",
         "openReactionMenu?",
+        "onClickMessageReplyAll?",
+        "onClickMessageForward?",
```

Comment:
Solution is not to pass props to `MessageActionMenuMobile`, because it has these drawbacks:
- forces `Message` -> `MessageActionMenuMobile` relationship
- If another component want to use `message_actions`, this requires doing some shenanigans to pass similar functions in shape and behaviour

The correct solution is to put `onClickMessageReplyAll` and `onClickMessageForward` code inside the message actions themselves: have it call a loose function that all message_actions user just work as long as they have `component.props.message` and `component.props.thread` _(and `component.env.services` but all components have it)_.